client/angular/controllers/MainController.js

```diff
@@ -51,9 +51,19 @@
         };
 
         // Ordering function such that newer tweets precede older tweets
+        // Pinned tweets are always placed before unpinned tweets
         var chronologicalOrdering = function(tweetA, tweetB) {
-            return new Date(tweetB.created_at).getTime() - new Date(tweetA.created_at).getTime();
```

Comment:
I think it would be cleaner here to write a separate function for the pinned tweets column, since each column is passed its own ordering function.